src/frontend/e2e/setup/global-setup.ts

```diff
@@ -1,14 +1,16 @@
 import { chromium, FullConfig } from "@playwright/test";
 import { getByID, getByTestID } from "../utils/selectors";
-import * as fs from "fs";
+import path from "path";
+import dotenv from 'dotenv';
 
+dotenv.config({path: path.resolve(`.env.${process.env.NODE_ENV}`),});
 
-const username = process.env.USERNAME ?? "";
-const password = process.env.PASSWORD ?? "";
+
+const username = process.env.USERNAME as string;
+const password = process.env.PASSWORD as string;
 
```

Comment:
can you do this instead? 
```suggestion
const username: string = process.env.USERNAME;
const password: string = process.env.PASSWORD;

```

Reply:
fixed

Reply:
The casting was to make TypeScript and IDE happy really